source/Ship.cpp

```diff
@@ -2347,10 +2348,14 @@ void Ship::DoGeneration()
 	else
 	{
 		// Ramscoops work much better when close to the system center.
+		// Carried fighters can't collect fuel or energy this way.
 		if(currentSystem)
 		{
 			double scale = .2 + 1.8 / (.001 * position.Length() + 1);
-			fuel += currentSystem->SolarWind() * .03 * scale * sqrt(attributes.Get("ramscoop"));
+			// If the universal ramscoop gamerule is true, then even if it ship has no ramscoop,
+			// it can harvest a tiny bit of fuel by flying close to the star.
+			double universal = 0.5 * scale * GameData::GetGamerules().UniversalRamscoopActive();
+			fuel += currentSystem->SolarWind() * .03 * scale * (sqrt(attributes.Get("ramscoop")) + universal);
```

Comment:
It's intentional that you're using a different formula for the universal ramscoop here right?

Reply:
No, just accidentally missed a 0.